src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Field.cs

```diff
@@ -65,7 +64,7 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
             if ((object)underlyingField == null)
             {
                 // Use-site error must have been reported elsewhere.
-                return new BoundFieldAccess(syntax, rewrittenReceiver, tupleField, constantValueOpt, resultKind, type, hasErrors: true);
+                return _factory.Field(rewrittenReceiver, tupleField);
```

Comment:
It would work too.

---
In reply to: [66331167](https://github.com/dotnet/roslyn/pull/11681#discussion_r66331167) [](ancestors = 66331167)